engine/config.go

```diff
@@ -178,6 +178,7 @@ func configBootstrap(args []string) Configuration {
 							Identifier:  "cdn-storage-local",
 							LocatorSalt: sdk.RandomString(9),
 							SecretValue: sdk.RandomString(17),
+							Timestamp:   time.Now().Unix(),
```

Comment:
Should be really set this value, does it means the end of key validity or the beginning?

Reply:
We also see it as a creation date
By setting a non nul timestamp, we are documenting the date of the creation of the key, which is a requirement.

In fact symmecrypt use it to order the keys.

Reply:
Ok good